tsconfig.test.json

```diff
@@ -1,6 +1,22 @@
 {
-  "extends": "./tsconfig.base",
   "compilerOptions": {
-    "types": ["node", "jest"],
-  }
+    "composite": true
+  },
+  "files": [],
+  "include": [],
+  "references": [
+    { "path": "./packages/apollo-cache-control/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-datasource-rest/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-engine-reporting/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-server/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-server-cache-memcached/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-server-cache-redis/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-server-caching/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-server-core/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-server-express/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-server-hapi/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-server-koa/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-server-lambda/src/__tests__/tsconfig.json" },
+    { "path": "./packages/apollo-server-micro/src/__tests__/tsconfig.json" },
```

Comment:
These _could_ be without `tsconfig.json`, right?

Reply:
Yep :)